packages/styles/scss/patterns/leadspace/_leadspace.scss

```diff
@@ -6,9 +6,7 @@
 //
 
 @import '../../globals/imports';
-@import 'carbon-components/scss/components/button/button';
-
-$btn-min-width: 26;
```

Comment:
We need this variable, each button has a `min-width: carbon--mini-units($btn-min-width);`